Userland/Libraries/LibGfx/ImageFormats/WebPSharedLossless.h

```diff
@@ -39,6 +39,14 @@ class CanonicalCode {
     Variant<u32, Compress::CanonicalCode> m_code;
 };
 
+ALWAYS_INLINE ErrorOr<void> CanonicalCode::write_symbol(LittleEndianOutputBitStream& bit_stream, u32 symbol) const
+{
+    TRY(m_code.visit(
+        [&](u32 single_code) __attribute__((always_inline))->ErrorOr<void> { VERIFY(symbol == single_code); return {}; },
```

Comment:
I'm surprised `clang-format` doesn't complain about the lack of spaces around `->`

Reply:
It in fact removed the spaces I had put there. I'm guessing putting attributes there just isn't very common. I filed https://github.com/llvm/llvm-project/issues/92657 upstream for this.